### PR TITLE
Refactor wordsmith updates

### PR DESCRIPTION
**Pull request by elisabethnewell**

### CODE REVIEW

File: content/tutorials/proptypes.md [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
+In this 15 minute tutorial, you will learn how to create a block with configurable light and dark modes.
 
 ![Preview](result-preview.png)
 
 ## Prerequisites
 
-- [Basic understanding of blocks](explanations/element-concepts)
-- [Element CLI installed](/how-to/env-setup)
+- [Element Concepts](/explanations/element-concepts)
+- [Element Setup](/how-to/env-setup)
```
Comment (chris-decker-volusion) [inline content/tutorials/proptypes.md]:
```suggestion
- [Element CLI Setup](/how-to/env-setup)
```

Comment (elisabethnewell) [reply content/tutorials/proptypes.md]: I'll change the title of that how-to page as well. Would "Environment Setup" or "Element CLI Setup" be more appropriate?

Comment (chris-decker-volusion) [reply content/tutorials/proptypes.md]: You know what, scratch that. After reviewing the file again, I think what you have is fine.

**Comment by chris-decker-volusion** [comment]
Deploying.

**Pull request merged**